couchexport/groupexports.py

```diff
@@ -49,6 +49,7 @@ def rebuild_export(config, schema, output_dir, last_access_cutoff=None):
             else:
                 saved.configuration = config
             saved.last_updated = datetime.utcnow()
+            saved.last_accessed = datetime.utcnow()
```

Comment:
i might be missing something, but it looks like this would set it every time? I would think you would put this line in the `if not saved` condition or add a check for if `last_accessed` was `None`